apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerClientCreationOptions.cs

```diff
@@ -30,106 +27,102 @@ namespace Google.Cloud.Spanner.Data
     /// </summary>
     internal sealed class SpannerClientCreationOptions : IEquatable<SpannerClientCreationOptions>
     {
-        private static readonly Lazy<Task<ChannelCredentials>> s_defaultCredentialsTaskProvider = new Lazy<Task<ChannelCredentials>>(CreatedScopedDefaultCredentials);
-
-        private static async Task<ChannelCredentials> CreatedScopedDefaultCredentials()
+        private static Lazy<DatabaseAdminSettings> DatabaseAdminSettingsCache { get; } = new Lazy<DatabaseAdminSettings>(CreateDatabaseAdminSettings);
```

Comment:
Why does this need to be lazy at all? Can't we just create it immediately? (We used to have laziness to avoid acquiring default credentials eagerly, but that's not in DatabaseAdminSettings.)

Reply:
Yep, done.